examples/draw_turtle.rs

```diff
@@ -0,0 +1,314 @@
+extern crate turtle;
+
+use turtle::Turtle;
+
+const SIZE: f64 = 1.0;
+const FIRST_COLOR: [f64; 4] = [62.0, 114.0, 29.0, 1.0];
+const SECOND_COLOR: [f64; 4] = [119.0, 178.0, 85.0, 1.0];
+const EYE_COLOR: &str = "black";
+
+fn main() {
+    let mut turtle = Turtle::new();
+    turtle.set_speed(8);
+
+    turtle.pen_up();
+    turtle.set_x(-280.0);
+    turtle.set_y(-90.0);
+
+    draw_panther(&mut turtle);
+
+    draw_tail(&mut turtle);
+
+    turtle.pen_up();
+    turtle.right(55.0);
+    turtle.forward(SIZE * 10.0);
+    turtle.right(55.0);
+
+    // Back leg
+    draw_leg(&mut turtle);
+
+    turtle.right(86.5);
+    turtle.forward(SIZE * 55.0);
+    turtle.right(86.0);
+
+    // Front leg
+    draw_leg(&mut turtle);
+
+    turtle.right(54.5);
+    turtle.forward(SIZE * 8.0);
+
+    draw_neck(&mut turtle);
+
+    turtle.right(172.6);
+    turtle.forward(SIZE * 40.0);
+    turtle.right(110.0);
+
+    draw_head(&mut turtle);
+
+    turtle.left(128.0);
+    turtle.forward(SIZE * 15.0);
+
+    draw_eye(&mut turtle);
+
+    turtle.set_fill_color(SECOND_COLOR);
+
+    turtle.left(175.0);
+    turtle.forward(SIZE * 43.0);
+
+    draw_right_highlight(&mut turtle);
+
+    turtle.right(18.0);
+    turtle.forward(SIZE * 37.0);
+    turtle.set_heading(180.0);
+
+    draw_middle_highlight(&mut turtle);
+
+    turtle.left(24.0);
+    turtle.forward(SIZE * 36.0);
+    turtle.set_heading(180.0);
+
+    draw_left_highlight(&mut turtle);
+}
+
+fn draw_panther(turtle: &mut Turtle) {
```

Comment:
Is there a reason this is called `draw_panther`? I think this method draws the turtle's shell.

Reply:
Funny mistake. On my native language the *panz(th)er* is a *shell* on English.